tests/test_address.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+* @brief 
+* @file test_crpyto.cpp
+* @author Mohit Agnihotri 
+* @date 2018-10-27
+*/
+
+/* module header */
+
+/* system includes C */
+#include <stdint.h>
+
+/* system includes C++ */
+#include <gtest/gtest.h>
+
+/* local includes */
+#include <helpers/hextobin.h>
+#include <eth/address.h>
+#include <eth/sign.h>
+
+TEST(TEST_ADDRESS, TEST_ADDRESS_RECOVERY)
```

Comment:
I think this test would also fit in the `test_crypto` file, what do you think?

Reply:
I agree we can add it to the same file and remove the new file altogether.